src/metadata/selectors.js

```diff
@@ -66,6 +66,31 @@ export const getCategoryOptionCombosByCategoryComboId = (
  * selectors that should have a cache per parameter (say an id) should use re-reselect.
  */
 
+export const getCategoryOptionsByCategoryOptionComboId = createCachedSelector(
+    (metadata) => metadata,
+    (_, categoryOptionComboId) => categoryOptionComboId,
+    (metadata, categoryOptionComboId) => {
+        const categoryCombos = Object.values(metadata.categoryCombos)
+
+        // "outer:" is a positional label that can be used to break multiple
+        // loops: https://stackoverflow.com/a/1564838
```

Comment:
Oops, in my suggested commit forgot I forgot to remove the comment which is now redundant.